packages/Card/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@paprika/card",
+  "version": "0.1.0",
+  "description": "The Card component is a customizable content container.",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika.git",
+    "directory": "packages/Card"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/helpers": "^0.2.11",
+    "@paprika/stylers": "^0.2.7",
+    "@paprika/tokens": "^0.1.14",
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "styled-components": "^4.2.0"
```

Comment:
Do we have any l10n component need from aria perspective @oscarkwan ? if we do let's add L10n as peer if not ignore my comment.